syllabus/grading.md

```diff
@@ -279,11 +279,13 @@ Community Badge values:
 
 
 You can earn community badges by: 
-- fixing small issues on the course website (during  only)
+- fixing small issues on the course website (during penalty free zone only)
 - contributing extra terms or reviews to your team repo 
 - sharing articles and discussing them in the course discussions 
 - contributing annotated resources the course website 
 
+You will maintain a list of your contributions in your KWL repo in the `community_contributions.md` file. All changes to this file should be submitted as a PR, with a review requested from @brownsarahm. 
```

Comment:
You will maintain a list of your contributions in your KWL repo in the community_contributions.md file. Every individual change to this file (representing one contribution) should be commited to a new branch and then submitted as a PR, with a review requested from @brownsarahm.

Reply:
You can make this a suggestion using the add a suggestion button ()
![an icon a page with +- on it](https://github.com/introcompsys/fall2023/assets/10656079/747b2cfb-7f5e-486d-a945-2b459ce49ece)

(using that feature is the second badge worthy actitivy)